packages/react-native/ReactAndroid/src/main/java/com/facebook/react/bridge/BaseActivityEventListener.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.react.bridge
+
+import android.app.Activity
+import android.content.Intent
+
+/** An empty implementation of [ActivityEventListener]. */
+public class BaseActivityEventListener : ActivityEventListener {
+  @Deprecated("Use onActivityResult(Activity, Int, Int, Intent) instead.")
```

Comment:
Could you add a `@ReplaceWith` here also?

Reply:
Added in cd316b17498a8e26113592e05250f4c59338ed7e